src/routers/handlers/extension-already-submitted/extensionAlreadySubmittedHandler.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response } from "express";
+import { BaseViewData, GenericHandler, ViewModel } from "../generic";
+import logger from "../../../lib/logger";
+import { servicePathPrefix, Urls } from "../../../lib/constants";
+
+export class ExtensionAlreadySubmittedHandler extends GenericHandler<BaseViewData> {
+
+    public static templatePath = "router_views/extension-already-submitted/extension-already-submitted";
+
+    public async getViewData (req: Request, res: Response): Promise<BaseViewData> {
+        const baseViewData = await super.getViewData(req, res);
+
+        return {
+            ...baseViewData,
+            backURL: servicePathPrefix + Urls.EXTENSION_CONFIRMATION,
+            templateName: Urls.EXTENSION_ALREADY_SUBMITTED
```

Comment:
This reads more intuitive as we can see the service path prefix is constant and after the path prefix, is the actual path

```suggestion
            backURL: SERVICE_PATH_PREFIX + PATHS.EXTENSION_CONFIRMATION,
            templateName: PATHS.EXTENSION_ALREADY_SUBMITTED
```